linear-algebra/eigenvalue-problem.md

```diff
@@ -1 +1,154 @@
 # Eigenvalue problem
+
+Find the eigenvalues and eigenvectors of the following matrices:
+
+1. \begin{equation}
+\vv{A} = \begin{bmatrix} 4 & 2 \\ 0 & -4 \end{bmatrix}
+\end{equation}
+
+```{solution}
+Since A is **upper triangular**, the eigenvalues are the diagonal entries:
+
+\begin{align}
+\lambda_1 &= 4 \\ 
+\lambda_2 &= -4
+\end{align}
+
+Then, use these eigenvalues to find the eigenvectors:
+
+\begin{equation}
+\lambda_1 = 4: \begin{bmatrix} 0 & 2 \\ 0 & -8 \end{bmatrix} x_1 = 0 
+\longrightarrow
+x_1 = \begin{bmatrix} 1 \\ 0 \end{bmatrix}
+\end{equation}
+
+\begin{equation}
+\lambda_2 = -4: \begin{bmatrix} 8 & 2 \\ 0 & 0 \end{bmatrix} x_2 = 0 
+\longrightarrow
+x_2 = \begin{bmatrix} 1 \\ -4 \end{bmatrix}
+\end{equation}
+
+So.
+\begin{align}
+\lambda_1 &= 4 \\
+x_1 &= \begin{bmatrix} 1 \\ 0 \end{bmatrix} \\
+\lambda_2 &= -4 \\
+x_2 &= \begin{bmatrix} 1 \\ -4 \end{bmatrix}
+\end{align}
```

Comment:
```suggestion
\begin{align}
\vv{x_1} &= \begin{bmatrix} 1 \\ 0 \end{bmatrix}&\lambda_1 &=4\\
\vv{x_2} &= \begin{bmatrix} 1 \\ -4 \end{bmatrix}&\lambda_2 &=-4
\end{align}
```

This would look nicer if the eigenvalues and eigenvectors were grouped together like in the notes